cmd/kube-apiserver/app/server.go

```diff
@@ -68,6 +68,7 @@ type APIServer struct {
 	BindAddress                net.IP
 	AdvertiseAddress           net.IP
 	SecurePort                 int
+	ReadWritePort              int
```

Comment:
As explained in another comment I can try to reverse engineer this from other flags - use secure port if we expose it, otherwise insecure one.